src/components/TitleSubtitle/TitleSubtitle.jsx

```diff
@@ -0,0 +1,25 @@
+import './TitleSubtitle.css';
+
+// 1. Alle html binnen de <></>
+// 2. class mag niet, het is className
+
+function TitleSubtitle({title, subTitle, className}) {
+    return (
+        <>
+            <div>
+
+                <h2 className={className}>
+
+                    <span><strong>{title}</strong></span>
+                </h2>
+
+                <p className={className}>
```

Comment:
Goed opgelost zo met de properties, chapeau! En dat terwijl je nog nooit met React hebt gewerkt 👍  Er is nog een iets geavanceerdere manier om dit te doen, dus mocht je nog een keertje langskomen bij Coding Wednesday, dan kunnen we daar nog eens naar kijken!